[stable/gcp-night-king] Add new Helm Chart for GCP Night King service

itamaro opened the pull request:
Did my best following contribution guidelines & best practices :-)

The service behind the Chart: https://github.com/itamaro/gcp-go-night-king

itamaro committed:
[stable/gcp-night-king] Add new Helm Chart for GCP Night King service

itamaro commented:
/assign @scottrigby

unguiculus commented:
Please make list indentation consistent. I'd suggest you always indent by two spaces.

itamaro commented:
> Please make list indentation consistent. I'd suggest you always indent by two spaces.

fixed

unguiculus commented on stable/gcp-night-king/templates/_helpers.tpl:
```diff
@@ -0,0 +1,16 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "nightking.name" -}}
```
The template namespace should be the same a the chart name: `gcp-night-king`.

itamaro replied:
right. fixed.

itamaro commented:
Any other concerns I should address?

unguiculus commented:
/ok-to-test

itamaro commented:
Thanks for approving this @unguiculus :-)

I've squashed and rebased, what else do I need to do to get this merged?

I don't know what it means "Not in tide pool."...

unguiculus commented:
You didn't have to squash. That's done upon merge anyways. Only my lgtm is missing.

unguiculus commented:
/lgtm

itamaro commented:
> You didn't have to squash. That's done upon merge anyways. Only my lgtm is missing.

cool, thanks :-)
